### PR TITLE
Prepare for release v0.6.3

### PR DESCRIPTION
**Pull request by TG1999**
Add CHANGELOG and bump tool version

Also bump click in setup.cfg
Reference: https://github.com/nexB/python-inspector/issues/54

Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -55,7 +55,7 @@ python_requires = >=3.6.*
 
 install_requires =
     attrs >= 18.1, !=20.1.0
-    click >= 6.7, !=7.0
+    click >= 8.1.3, !=7.0
```
Comment (pombredanne) [inline setup.cfg:58]: Why do you need Click 8.1.3 + ?

Comment (TG1999) [reply setup.cfg:58]: Because of #54 , I have referred same in commit message as well.

**Review by pombredanne — CHANGES_REQUESTED** [review]
We need to relax the deps for Click

**Review by pombredanne — APPROVED** [review]
LGTM!
Thanks.

**Pull request merged**